src/twisted/protocols/ftp.py

```diff
@@ -667,7 +667,7 @@ def __init__(self, cons):
             self.write = cons.write
```

Comment:
This conditional is wrong too

Reply:
I dont think this conditional is wrong, `os.linesep` return str type, and so is "\r\n", no need to convert to bytes here.